src/Opdex.Platform.Common/Extensions/ListExtensions.cs

```diff
@@ -1,23 +0,0 @@
-using System.Collections.Generic;
-using System.Linq;
-
-namespace Opdex.Platform.Common.Extensions
-{
-    public static class ListExtensions
-    {
-        /// <summary>
-        /// Split an IEnumerable into multiple smaller chunks.
-        /// </summary>
-        /// <param name="items">The items to split up.</param>
-        /// <param name="itemsPerChunk">The number of items per chunked list.</param>
-        /// <typeparam name="T">The type of IEnumerable being chunked.</typeparam>
-        /// <returns>2 dimensional array of chunked lists.</returns>
-        public static IEnumerable<IEnumerable<T>> Chunk<T>(this IEnumerable<T> items, int itemsPerChunk)
-        {
```

Comment:
We use this is a few areas, curious how removing this works?

Reply:
.NET 6 LINQ has this method natively https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.chunk?view=net-6.0